src/main/java/uk/gov/pay/api/utils/TrustStoreLoader.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.api.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+public class TrustStoreLoader {
```

Comment:
Commit is marked as experimental for a reason!

I don't follow what you mean by "remove the certs after a successful load.". The certs may change between container restarts and we need to notice this and pick up the new certs when they change.